components/lib/stepper/StepperBase.js

```diff
@@ -0,0 +1,116 @@
+import { ComponentBase } from '../componentbase/ComponentBase';
+import { classNames } from '../utils/Utils';
+
+const classes = {
+    root: ({ props }) =>
+        classNames('p-stepper p-component', {
+            'p-stepper-horizontal': props.orientation === 'horizontal',
+            'p-stepper-vertical': props.orientation === 'vertical',
+            'p-readonly': props.linear
+        }),
+    nav: 'p-stepper-nav',
+    stepper: {
+        header: ({ isStepActive, isItemDisabled, index }) => {
+            return classNames('p-stepper-header', {
+                'p-highlight': isStepActive(index),
+                'p-disabled': isItemDisabled(index)
+            });
+        },
+        action: 'p-stepper-action',
+        number: 'p-stepper-number',
+        title: 'p-stepper-title',
+        separator: 'p-stepper-separator',
+        toggleableContent: 'p-stepper-toggleable-content',
+        content: ({ props }) =>
+            classNames('p-stepper-content', {
+                'p-toggleable-content': props.orientation === 'vertical'
+            })
+    },
+    panelContainer: 'p-stepper-panels',
+    panel: ({ props, isStepActive, index }) =>
+        classNames('p-stepper-panel', {
+            'p-stepper-panel-active': props.orientation === 'vertical' && isStepActive(index)
+        })
+};
+
+const styles = `
```

Comment:
i believe we can move this styles to react sass theme

Reply:
I will do it during the theme update, thanks.